tests/causal_refuters/test_add_unobserved_common_cause.py

```diff
@@ -71,4 +72,36 @@ def test_refutation_continuous_treatment_range_outcome(self, mock_fig, error_tol
                 effect_strength_on_y = effect_strength_on_y)
         refuter_tester.continuous_treatment_testsuite(tests_to_run="atleast-one-common-cause")
         assert mock_fig.call_count > 0  # we patched figure plotting call to avoid drawing plots during tests
+    @pytest.mark.parametrize(["estimator_method", "effect_strength_on_t", "benchmark_covariates", "simulated_method_name"],
+                             [("backdoor.linear_regression", [1,2,3], ["W3"], "PartialR2"),])
+    @patch("matplotlib.pyplot.figure")
+    def test_linear_sensitivity_analysis(self, mock_fig,estimator_method,
```

Comment:
for the tests, we can test a few more things with the same function, 
1. if increase=True, then the adjusted estimate should be lower than original estimate.
2. check that the partial r2 values are all between 0 and 1.

and add more functions to test:
1. create two settings. If partial R2 of both t and y is higher in one setting, then its adjusted estimate should be farther from original estimate compared to the original estimate.
2. create a simple dataset without any unobserved confounding. test: its robustness value should be higher than a fixed threshold, say 0.95.